program/c/src/oracle/oracle.c

```diff
@@ -511,6 +512,9 @@ static uint64_t upd_price( SolParameters *prm, SolAccountInfo *ka )
     fptr->status_   = status;
     fptr->pub_slot_ = cptr->pub_slot_;
   }
+  if (updated_aggregate){
+    return SUCCESSFULLY_UPDATED_AGGREGATE;
+  }
   return SUCCESS;
 }
```

Comment:
You need to Update upd_price_no_fail as well

Reply:
Done